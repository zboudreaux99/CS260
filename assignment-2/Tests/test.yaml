test:
  timelimit: 10
  launch: ./tally
  params: '@T/albums.csv @T/requests.csv'
  stderr: false
dueDate: 12/06/2024
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["playlist.h", "playlist.cpp"]
  instructorFiles:
    exclude: ["playlist.h", "playlist.cpp"]
