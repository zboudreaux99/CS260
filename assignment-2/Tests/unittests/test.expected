1..3
# Warning: No matching test found for input specification /home/zeil/courses/cs260/f24/Assts/class_playlist/Grading/unittests/albums.csv
# Warning: No matching test found for input specification /home/zeil/courses/cs260/f24/Assts/class_playlist/Grading/unittests/requests.csv
ok 1 - Add1
ok 2 - AddMulti
ok 3 - Constructor
# UnitTest: passed 3 out of 3 tests, for a success rate of 100.0%
